src/transformers/AddSources.ts

```diff
@@ -32,10 +32,10 @@ async function addSourcesContent (sourceMapPath: string, map: UnsafeSourceMap, p
       let source = null
       try {
         // don't look up sources for virtual webpack files
-        if (!/^webpack:\/\/\/webpack/.test(p)) {
+        if (!/^webpack:\/\/(.*)\/webpack/.test(p)) {
           const absoluteSourcePath = path.resolve(
             path.dirname(sourceMapPath),
-            p.replace(/webpack:\/\/\/\.\//, `${projectRoot}/`)
+            p.replace(/webpack:\/\/.*\/\.\//, `${projectRoot}/`)
```

Comment:
slightly curious as to how this hasn't impacted the e2e tests. Prior to this change the `sourcesContent`was already being populated, so this doesn't seem to have any effect in those tests

Reply:
The Webpack fixtures are configured to use `devtool: source-map`, which includes the `sourceContent` in the generated maps

We could add a new fixture with `devtool: nosources-source-map`, which should show that this works. Editing an existing fixture would also work, but that's slightly more complicated because Webpack includes the source of its virtual files which isn't something we can do